Information about Dedicated Generation 3 (#1535)
platformsh/platformsh-docs: docs/src/overview/dedication.md

---
title: "Dedication Production Clusters"
weight: 1
---

> The Dedication Production Clusters is astonishing if you want a mature and extensive application and scalable solution in the cloud.

Reviewer: ```suggestion
Platform.sh Dedication Production Clusters provide a scalable solution as an additional option on top of your existing Grid applications.
```

> It provides a set of Virtual Machine redundant configuration,  a minimum of three instances,  provisioned by Platform.sh for each customer.  Every service is replicated across all three virtual machines in a failover configuration (as opposed to sharding), allowing a site to remain up even if one of the VMs is lost entirely.

Reviewer: ```suggestion
It provides redundant configuration, a minimum of three Virtual Machine instances, provisioned by Platform.sh for each customer. Every service is replicated across all three virtual machines in a failover configuration (as opposed to sharding), allowing a site to remain up even if one of the VMs is lost entirely.
```

> Dedicated Production Clusters work nearly identically to Grid environments, and it does not require extra configuration on the client-side.  However, the services PostgreSQL or Kafka are not available on this plan.

Reviewer: ```suggestion
Dedicated Production Clusters work nearly identically to Grid environments, and does not require additional  configuration on your part. The only difference is that of service availability - the services PostgreSQL or Kafka are not available on Dedication.
```

## Features

> * Domains available to dev environments

Reviewer: ```suggestion
* Domains can be added to your development environments, such as to a dedicated staging environment.
```

* The master branch with three-Virtual Machine
* Support to Data sync
* Backup on Production does not freeze in the process.
* You can deploy your application in any of our [supported cloud providers](development/faq.md#which-geographic-zones-does-platformsh-cover). 

> You cannot perform the following:

Reviewer: Seems repeated. I would choose to include this in the intro  or here, and it makes a little more sense to me as a list here for some reason.

Author: I'll keep the information here.

* Use PostgreSQL or Kafka (The support for those is upcoming).

> ## Enter a ticket for updating the Project to Dedication Production Clusters

Reviewer: ```suggestion
## Setting up
```

> Enter a Support ticket with the suggested title “Upgrade to Dedication Production Clusters. In the ticket, request to have your project Upgrade.

Reviewer: ```suggestion
Open a Support ticket with the suggested title “Upgrade to Dedication Production Clusters". In the ticket, request to have your project Upgrade.
```

We will review the infrastructure and settings, create user and environment variables for Staging and Production environments, and update the ticket with results.

Once started the process usually takes less than an hour.  There should be no downtime on your production site, although you should not push any code to Git while the upgrade is in progress.

> When done, you can access review your project through the Project Web Interface.

Reviewer: ```suggestion
When done, you can review the changes to your project through the management console.
```
